net/loadbalancer.md

```diff
@@ -0,0 +1,77 @@
+The purpose of this document is to demonstrate the current implementation of service loadblancer support for docker
+
+##What is a Service ? What are providers for a service
+A service is an abstration to provide a network connection  on a set of ports to a cluster of containers matching the service 
+label-selector. Services are easy to scale up/down. Services provide efficient loadbalancing and minimal downtime. 
+A provider can be one or many containers that match the label selector associated with the services. 
+
+##How is a Service defined
+
+Current implementation allows services to be defined using netctl. Services need to be created in a service network. So we create a service network and attach  the service to the service network
+```
+netctl net create cisco-srv-net -s 100.1.1.0/24
+
+netctl service create app-svc --network cisco-srv-net --tenant default --selector=app=myapp --port 8080:80:TCP
```

Comment:
if `tenant` is not required, can you avoid it in the example. Also `selector` keyword is worth explaining a bit more.